docs/managing-dragonfly/using-http.md

```diff
@@ -0,0 +1,24 @@
+# HTTP
+
+Dragonfly supports HTTP for certain operations. By default, `HTTP` can be accessed on 
```

Comment:
It should be `https` in that case but I don't see any tests around it. I think it should work outside of the box based on the code I see in dragonfly_connection.cc. I did not add it here, since TLS is a separate concept which we discuss in `using-tls.md` (and not in other places)